vdgsa_backend/rental_viols/templates/users/move.html

```diff
@@ -0,0 +1,33 @@
+{% extends '../rentals.html' %}
+{% load rental_tags %}
+{% block content %}
+
+
+
+<div class="container border mt-3 mb-3 p-3">
+  <div class="row">
+    <div class="col">
+      <p>Move {{user}}</p>
+    </div>
+  </div>
+</div>
+<div class="">NOT IMPLEMENTED</div>
```

Comment:
Just checking that this is an intentional placeholder

Reply:
Feature was not implemented - this has been dropped.